orttraining/orttraining/python/training/ortmodule.py

```diff
@@ -269,14 +280,14 @@ def backward(ctx, *grad_outputs):
         # Disable external allocator for ROCM EP since external allocator is not supported yet.
         self._use_external_cuda_allocator = (False if self.is_rocm_pytorch else True)
         if self._use_external_cuda_allocator:
-            self._torch_cuda_allocator = _load_torch_allocator_cpp_extension()
+            self._torch_cuda_allocator = _load_torch_allocator_cpp_extension(self._verbosity)
             self._torch_alloc = self._torch_cuda_allocator.cuda_caching_allocator_raw_alloc_address()
             self._torch_free = self._torch_cuda_allocator.cuda_caching_allocator_raw_delete_address()
 
     def _initialize_module_gradient_graph_builder(self):
         # TODO: PyTorch exporter bug: changes the initializer order in ONNX model
         initializer_names = [p[0] for p in self._flattened_output_module.named_parameters()]
-        onnx_initializer_names = [p.name for p in self._onnx_inference.graph.initializer]
+        onnx_initializer_names = {p.name for p in self._onnx_inference.graph.initializer}
```

Comment:
Why did it become a dict in this PR?

Reply:
I changed to a set since we only use it for lookups. and don't need it for any other purpose.